src/main/resources/assets/clientcommands/lang/en_us.json

```
@@ -249,6 +249,16 @@
   "commands.ctask.stop.noMatch": "No matching tasks",
   "commands.ctask.stop.success": "Stopped %s tasks",
 
+  "commands.ctick.client.mspt.expectedMspt": "The expected client MSPT is %s",
+  "commands.ctick.client.tps.expectedTps": "The expected client TPS is %s",
+  "commands.ctick.measuring": "Measuring...",
+  "commands.ctick.mspt": "Milliseconds per tick = %s",
+  "commands.ctick.mspt.inaccurate": "MSPT is inaccurate above 20TPS",
```

The TPS isn't necessarily 20 anymore.
```suggestion
  "commands.ctick.mspt.inaccurate": "MSPT is inaccurate when the server isn't lagging",
```